README.md

```diff
@@ -29,7 +29,7 @@ pip install shadowfinder
 If you want to use ShadowFinder directly from Python, the usage is as follows.
 
 ```python
-from shadowfinder.shadowfinder import ShadowFinder
+from shadowfinder import ShadowFinder
```

Comment:
I added `ShadowFinder` to `shadowfinder/__init__.py`, which allows us to import from the `shadowfinder/` module like it was a file. This makes it easier for users to import classes/functions/etc. without having to know which file those entities belongs to.

Reply:
Good eye. Updated the import in `ShadowFinderColab.ipynb` in ad92a31